refactor doc

gogo2464 opened the pull request:
I had also set a easier way to navigate to the doc. I will merge all in hugo as soon as possible.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/freenet/web?pullRequest=7) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/freenet/web?pullRequest=7) before we can accept your contribution.<br/><hr/>**test** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/freenet/web?pullRequest=7) it.</sub>

sanity commented:
What's the purpose of these changes? I thought you were working on migrating the user manual into this codebase.

gogo2464 commented:
@sanity  : 

I read the user manual as starting. It has took me a few times. I think the actual documentation of freenet is VERY confusing.

I am expert in clarifying documentation.

I realized that the doc of freenet is very confusing about the `ressource` tab. What is a ressource? Developper doc? user doc? Ressource on freenet coding context? history of freenet? As you told previously, we need to merge all doc in a single site and then in tabs. I had removed this tab to merge it in these tabs:

I split the doc  in 4 types:
- install tutorial
- real life usage tutorial
- developper api reference
- global explanation about the goal of the freenet dev and the freenet context where it has been built.

I want to make documentation navigation easier for everyone reading the web site.
Navigation will be easier to follow.

I am definitely sorry for the fact you dislike. I am supposed to shall make freenet dev easier and faster. Not harder and slower. I think it could be a great feature.


Best regards.

gogo2464 commented:
@sanity I see the lack of your interest in my way to solve it. I suggest that I:
1- fix the real issue by setting freenet-core doc to web doc.
2- once finished I will see for my point.

I should have been more organized and more incremental!!

sanity commented:
Closing this. We had agreed on a straightforward task: migrating the user manual markdown pages to the website. Instead, you significantly changed the site’s structure and design. That was over a month ago, and the original task still isn’t done. If you're not able to stay focused on the scope we agree on, then I don’t have the bandwidth to keep managing this.

gogo2464 commented:
Yes sure. Close. I will redo it better.
